Free_Games.py

```diff
@@ -113,16 +102,20 @@ def getGame(game):
             time.sleep(4)
             purchase = browser.find_element_by_xpath('''/html/body/div[3]/div/div/div[4]/div/div[4]/div[1]/div[2]/div[5]/div/div/button/span''')
             purchase.click()
+            time.sleep(2)
+            try:
+                browser.find_element_by_xpath('''//span[contains(text(),'I Agree')]''')
+            except:
+                print()
+            else:
+                EU_Refund_and_Right_of_Withdrawal_Information = browser.find_element_by_xpath('''//span[contains(text(),'I Agree')]''')
+                EU_Refund_and_Right_of_Withdrawal_Information.click()
+                time.sleep(2)
```

Comment:
This checks for a pop-up about the EU Refund and Right of Withdrawal Information and click I Agree